kube/services/jobs/aws-bucket-replicate-job.yaml

```diff
@@ -0,0 +1,92 @@
+# run with 
+# g3k runjob aws-bucket-sync \
+# GDC_BUCKET_NAME a_bucket_name \
+# CHUNK_SIZE chunk_size
+#
+# folow the below link  to setup bucket owner grant access to other aws account 
+# https://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example2.html
+#
+# setup bucket policy so that the user test can access to the bucket (GDC bucket)
+# {
+#     "Version": "2012-10-17",
+#     "Statement": [
+#         {
+#             "Sid": "Example permissions",
+#             "Effect": "Allow",
+#             "Principal": {
+#                 "AWS": "arn:aws:iam::xxxxxxxxxxx:user/test"
+#             },
+#             "Action": [
+#                 "s3:GetBucketLocation",
+#                 "s3:ListBucket"
+#             ],
+#             "Resource": "arn:aws:s3:::gdc_bucket"
+#         },
+#         {
+#             "Sid": "bucket permissions",
+#             "Effect": "Allow",
+#             "Principal": {
+#                 "AWS": "arn:aws:iam::xxxxxxxxxxx:user/test"
+#             },
+#             "Action": [
+#                 "s3:GetObject"
+#             ],
+#             "Resource": "arn:aws:s3:::gdc_bucket/*"
+#         }
+#     ]
+# }
+# 
+# HOW TO RUN
+# g3k runjob aws-bucket-replicate GDC_BUCKET_NAME gdc_test_bk_bucket LOG_BUCKET log_bucket CHUNK_SIZE 4
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-bucket-replicate
+spec:
+  # not yet supported - backOffLimit: 3
+  template:
+    metadata:
+      labels:
+        app: gen3job
+    spec:
+      volumes:
+        - name: cred-volume
+          secret:
+            secretName: "aws-creds-secret"
+        - name: manifest-volume
+          configMap:
+            name: data-service-manifest
+        - name: creds-json-volume
+          secret:
+            secretName: "dcf-dataservice-json-secret"
+      containers:
+      - name: awsreplicate
+        image: quay.io/cdis/dcf-dataservice:feat_data_replicate_service
+        imagePullPolicy: Always
+        env:
+          - name: GDC_BUCKET_NAME
+            GEN3_GDC_BUCKET_NAME
+          - name: LOG_BUCKET
+            GEN3_LOG_BUCKET
```

Comment:
What is the LOG_BUCKET for?

Reply:
To save all the uuid which can not be copied.